_pages/production.md

```diff
@@ -24,20 +24,20 @@ sidenav:
     href: "#certificate-rotation-process"
 saml_year: 2023
 ---
-<div class="usa-alert usa-alert--warning">
-<div class="usa-alert__body">
-<p class="usa-alert__text"><b> Please note: The deployment process can take up to two weeks to complete. You may wish to have a grace period between when we deploy and when you add the Login.gov button to your site. </b></p>
-</div>
-</div>
+{% capture deployment_warning %}
+Please note: The deployment process can take up to two weeks to complete. You may wish to have a grace period between when we deploy and when you add the Login.gov button to your site.
+{% endcapture %}
 
-Before deployment
------------------
+{% include alert.html content=deployment_warning alert_class="usa-alert--warning" %}
+
+## Before deployment
 
 Make sure you have the following items ready before you start the deployment process:
 
 -   [Signed Interagency Agreement (IAA) listing this integration ](https://developers.login.gov/production/#confirm-iaa)
 
--   A new dedicated [production app configuration within the Dashboard](https://dashboard.int.identitysandbox.gov/)
+-   A new dedicated [application configuration within the Dashboard](https://dashboard.int.identitysandbox.gov/)
```

Comment:
[nitpick] technically it doesn't need to be "new". what do you think about dropping that word?

Reply:
yeah I was on the fence about that myself - lets drop it 👍🏻